plugins/woocommerce/src/Internal/PushNotifications/Controllers/PushTokenRestController.php

```
@@ -57,15 +58,149 @@ protected function get_rest_api_namespace(): string {
 	}
 
 	/**
-	 * Get the schema for the POST endpoint.
+	 * Creates a push token record.
+	 *
+	 * @since 10.6.0
+	 *
+	 * @param WP_REST_Request $request The request object.
+	 * @return WP_REST_Response|WP_Error
+	 */
+	public function create( WP_REST_Request $request ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)